src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -186,7 +185,6 @@ export default React.memo(function GamePage(): JSX.Element | null {
 
         if (
           runner !== 'sideload' &&
-          !is_installed &&
```

Comment:
is this intentional? it's not clear why this is removed (I understand this makes sure the install info is not fetch when the game is already installed)